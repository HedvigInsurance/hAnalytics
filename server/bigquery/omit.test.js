const omit = require("./omit");

test("omits correctly", () => {
  expect(
    omit("property", {
      property: {
        hello: "hello",
      },
    })
  ).toMatchSnapshot();

  expect(
    omit("property", {
      property: {
        hello: "hello",
      },
      otherValue: "hello",
    })
  ).toMatchSnapshot();

  expect(
    omit(["property", "otherValue"], {
      property: {
        hello: "hello",
      },
      otherValue: "hello",
    })
  ).toMatchSnapshot();

  expect(
    omit(["property", "othersValue"], {
      property: {
        hello: "hello",
      },
      otherValue: "hello",
    })
  ).toMatchSnapshot();
});
