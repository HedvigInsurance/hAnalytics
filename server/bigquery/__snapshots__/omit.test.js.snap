// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`omits correctly 1`] = `Object {}`;

exports[`omits correctly 2`] = `
Object {
  "otherValue": "hello",
}
`;
